test/appframework_aws/c3/appframework_aws_test.go

```diff
@@ -909,26 +909,27 @@ var _ = Describe("c3appfw test", func() {
 			uploadedApps = append(uploadedApps, uploadedFiles...)
 
 			// Create App framework Spec
-			appSourceName := "appframework-shc-" + testenv.RandomDNSName(3)
+			appSourceNameShc := "appframework-shc-" + testenv.RandomDNSName(3)
 			appSourceVolumeNameShc := "appframework-test-volume-shc-" + testenv.RandomDNSName(3)
-			appFrameworkSpecShc := testenv.GenerateAppFrameworkSpec(ctx, testcaseEnvInst, appSourceVolumeNameShc, enterpriseApi.ScopePremiumApps, appSourceName, s3TestDirShc, 60)
-			appFrameworkSpecShc.Defaults.PremiumAppsProps = enterpriseApi.PremiumAppsProps{
+			appFrameworkSpecShc := testenv.GenerateAppFrameworkSpec(ctx, testcaseEnvInst, appSourceVolumeNameShc, enterpriseApi.ScopePremiumApps, appSourceNameShc, s3TestDirShc, 60)
+			appFrameworkSpecShc.AppSources[0].PremiumAppsProps = enterpriseApi.PremiumAppsProps{
+				Type: enterpriseApi.PremiumAppsTypeEs,
 				EsDefaults: enterpriseApi.EsDefaults{
-					SslEnablement: "ignore",
+					SslEnablement: enterpriseApi.SslEnablementIgnore,
 				},
 			}
 
 			// Deploy C3 SVA
-			// Deploy the Cluster Master
-			testcaseEnvInst.Log.Info("Deploy Cluster Manager")
-			_, err = deployment.DeployClusterMaster(ctx, deployment.GetName(), "", "", "")
-			Expect(err).To(Succeed(), "Unable to deploy Cluster Manager")
+			// // Deploy the Cluster Master
```

Comment:
Nit: quite a bunch of '// //' in this PR actually

Reply:
These are comments that should be taken care when we enable indexer Ideally